Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/pizza.yml

```
@@ -547,3 +547,73 @@
         - ReagentId: Vitamin
           Quantity: 1
 # Tastes like stale crust, rancid cheese, mushroom.
+
+- type: entity
+  name: spicy rock pizza
+  parent: FoodPizzaBase
+  id: FoodPizzaUranium
+  description: Spicy pizza covered in peppers and uranium.
+  components:
+  - type: FlavorProfile
+    flavors:
+    - spicy
+    - cheesy
+    - oily
+    - bread
+  - type: Sprite
+    layers:
+    - state: uranium-pizza
+  - type: SliceableFood
+    slice: FoodPizzaUraniumSlice
+  - type: Tag
+    tags:
+    - Meat
+    - Pizza
+  - type: PointLight
+    enabled: true
+    radius: 2
+
+  - type: SolutionContainerManager
+    solutions:
```

```suggestion
    enabled: true
    radius: 2
  - type: SolutionContainerManager
    solutions:
```
Remove the random space here.